.github/workflows/buildRelease.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       run: |
         cd $GITHUB_WORKSPACE
         python -m pip install wheel
+        TM_PACKAGE_VERSION=${{ github.event.inputs.python_package_version }}
+        printf "TORCH_MLIR_PYTHON_PACKAGE_VERSION=%s\n" $TM_PACKAGE_VERSION > ./tm_pkg_version
```

Comment:
can you call this torch_mlir_package_version. No need to abbreviate.

Reply:
done